ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh

```diff
@@ -943,14 +946,14 @@ function run_upgrade_e2e() {
     parallel_cucumber -n "${PARALLEL}" --first-is-1 --type cucumber --serialize-stdout --combine-stderr --prefix-output-with-test-env-number --exec \
     'export OPENSHIFT_ENV_OCP4_USER_MANAGER_USERS=$(echo ${USERS} | cut -d "," -f ${TEST_ENV_NUMBER},$((${TEST_ENV_NUMBER}+${PARALLEL})),$((${TEST_ENV_NUMBER}+${PARALLEL}*2)));
      export WORKSPACE=/tmp/dir${TEST_ENV_NUMBER};
-     parallel_cucumber --group-by found --only-group ${TEST_ENV_NUMBER} -o "--tags \"${E2E_RUN_TAGS} and not @admin\" -p junit"' || true
+     parallel_cucumber --group-by found --only-group ${TEST_ENV_NUMBER} -o "--tags \"${E2E_RUN_TAGS} and not @admin\" -p junit"' &>> "${test_log_dir}/4.${TARGET_MINOR_VERSION}-e2e-log.txt" || true
 
     # run admin tests
     export BUSHSLICER_REPORT_DIR="${ARTIFACT_DIR}/parallel-admin-${idx}"
     parallel_cucumber -n "${PARALLEL}" --first-is-1 --type cucumber --serialize-stdout --combine-stderr --prefix-output-with-test-env-number --exec \
     'export OPENSHIFT_ENV_OCP4_USER_MANAGER_USERS=$(echo ${USERS} | cut -d "," -f ${TEST_ENV_NUMBER},$((${TEST_ENV_NUMBER}+${PARALLEL})),$((${TEST_ENV_NUMBER}+${PARALLEL}*2)));
      export WORKSPACE=/tmp/dir${TEST_ENV_NUMBER};
-     parallel_cucumber --group-by found --only-group ${TEST_ENV_NUMBER} -o "--tags \"${E2E_RUN_TAGS} and @admin\" -p junit"' || true
+     parallel_cucumber --group-by found --only-group ${TEST_ENV_NUMBER} -o "--tags \"${E2E_RUN_TAGS} and @admin\" -p junit"' &>> "${test_log_dir}/4.${TARGET_MINOR_VERSION}-e2e-log.txt" || true
```

Comment:
Instead of customize partial of the e2e output (you can see that the echo statement and output from function filter_tests are still in the original log file), how about change `run_upgrade_e2e "${index}"` to `run_upgrade_e2e "${index}" &>> "${test_log_dir}/4.${TARGET_MINOR_VERSION}-e2e-log.txt` which will redirect all the e2e logs ?

Reply:
Yes, i intended to leave it in the build.log since it's kind of summary output, which will not disturb any more.

Reply:
How we filter e2e test cases are in the upgrade log, and e2e execution part are in separate logs. This make upgrade debug easier, but make e2e debug harder.

Reply:
ok, no preference on how e2e debugger work here, fine with me to move all the e2e logs out.